barretenberg/cpp/src/barretenberg/proof_system/execution_trace/execution_trace.cpp

```diff
@@ -6,28 +6,32 @@
 namespace bb {
 
 template <class Flavor>
-void ExecutionTrace_<Flavor>::generate(const Builder& builder,
+void ExecutionTrace_<Flavor>::generate(Builder& builder,
```

Comment:
This is just an imho, feel free to ignore. To me "generate" felt a bit misleading, because we are not generating the execution trace, we are generating a part of the proving key, which is one of the arguments.

Reply:
Good point. I'll change to "populate"